eZ/Publish/Core/Repository/ContentService.php

```diff
@@ -1187,7 +1187,16 @@ public function updateContent(APIVersionInfo $versionInfo, APIContentUpdateStruc
         }
 
         $mainLanguageCode = $content->contentInfo->mainLanguageCode;
-        $languageCodes = $this->getLanguageCodesForUpdate($contentUpdateStruct, $content);
+        if ($contentUpdateStruct->initialLanguageCode === null) {
+            $contentUpdateStruct->initialLanguageCode = $mainLanguageCode;
+        }
+
+        $allLanguageCodes = $this->getLanguageCodesForUpdate($contentUpdateStruct, $content);
+        foreach ($allLanguageCodes as $languageCode) {
+            $this->persistenceHandler->contentLanguageHandler()->loadByLanguageCode($languageCode);
```

Comment:
Seems to be purely for validation of the language codes, this was not done in the past was it?